### PR TITLE
Make Team pages more mobile friendly (#2217)

### LINKED ISSUE #2217: Feature Request: Make Team pages more mobile friendly
## Current Behavior
Some improvements are possible to team pages on phones.

image

## Possible Solution
1. Rename "Media" -> "Photos & Videos" in the left nav, and remove the sub-items. It's very rare you'd want to see one without the other, and this makes the page 2 items shorter.
2. Remove the "like button" "google+ button" and "Live Data: At the Control" row – these are almost never used.
3. Hide "Add CAD Model!" button that appears on top of page to make page shorter.
4. Similar change to actual "Media" section as to the left-nav to remove headers and make page shorter.

## Steps to Reproduce (for bugs)
Visit a team page like https://www.thebluealliance.com/team/5507 on a narrow browser

### PR DESCRIPTION
**Pull request by pordonj**
## Description

* "Collapsed" separate photo/video sections on the team page and nav
  * The presentation of the photos and video is still the same, but I added one extra "if" that shortly handles having no photos or videos (see screenshots below). If one or the other is present it will handle it as before.
  * What is the best way to test things like this? I added a suggestion on my local copy, and logged in a different account as administrator. I found the admin page but the link the Media suggestions didn't work (http://localhost:8080/admin/suggestions/media/review 404'd)
* Removed chunk of page that held FB Like, G+, and At the Control buttons
  * the team_social_media partial only created the FB Like and G+ buttons so I deleted it.
* Hid the "Add CAD Model" button near the top of the page if it would appear (The thing that would appear if there was already a CAD model will still appear, though)
* ~(Unrelated) Could I get an invite to the TBA Slack?~ Got one, thanks!

## Motivation and Context

Fixes #2217 

## How Has This Been Tested?

* Viewed in local docker container (see screenshots)
* Ran `paver test` in the docker container
* Ran `paver lint` but it didn't seem to do anything but after looking I think this only applies to python file changes?

## Screenshots (if appropriate):
Nav Before:
nav-before

Nav After:
navafter

Social Media/Upper Add CAD Before:
socialmedia-cad-before

Social Media/Upper Add CAD After:
socialmedia-cad-after

Media Section with no photos OR videos Before:
no-photo-or-video-before

Media Section with no photos OR videos After:
no-photo-or-video-after

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

**Comment by gregmarra** [comment]
Thanks for taking this on! I'm excited about how this will clean up these pages and appreciate your contribution to TBA!

@pordonj , in the future, it might be easier to submit 3 separate pull requests in separate branches for each of the 3 logical things you cleaned up here (collapsing media, removing social media buttons, removing the CAD button). Each pull would be smaller and easier to review. In this case, I would have been able to accept 2 of them immediately, since my comments only affected 1 of the 3 things you changed. These changes are pretty simple, so not a big deal here, but might help with larger future changes.

Please address the comment about ampersand encoding, and clean up the indentation style to match the rest of the file. @fangeugene do you have any additional comments? Otherwise I can merge this once the comments are addressed.

Also I've sent you a Slack invite :)

**Comment by bdaroz** [comment]
@gregmarra I think your comment on the & encoding got lost or didn't save.

**Comment by fangeugene** [comment]
Looks good to me. @gregmarra does your suggestion in #2217 to hide the social media like buttons apply to mobile only, or do you think they should be gone entirely? This current PR removes them entirely.

**Comment by bdaroz** [comment]
Just a thought, IRT to the G+/FB stuff... First, who the heck still uses G+? Second, I think the +1/Like for them applies to the entire site, not just the team page. You know... b/c 254's team page has like 24K likes, and so does my 5881 team too! We're just as popular! 😄 

Personally, I'd be for pulling them both from Team/Event detail pages as what you're liking (actual) doesn't correspond to quite what you're looking at. That and... well FB's thing lately, but I digress.

**Comment by gregmarra** [comment]
@fangeugene  the current like button likes the page on The Blue Alliance, but that doesn't generate anything visible on Facebook anymore, so my proposal is we remove it entirely.

In a separate feature request, we could directly include the button to follow a Team Page on Facebook, but that's not what the current button did.

**Comment by pordonj** [comment]
@gregmarra What should I do for the & encoding?

Also, I can't see what's off with the indentation? (2 spaces, right?) There is a bunch of stuff showing in the diff because a few chunks of code got indented 1 level in with my new "overall" photo/video if-check.

I will compartmentalize my PRs more in the future :) I fiddled with contributing back in 2013 and I think I want to get more into it. (Once I got it all working) the whole Docker setup made it really easy to edit and test everything!

File: templates_jinja2/team_details.html [hunk 1]
```diff
@@ -42,11 +42,7 @@ <h2 class="visible-xs end_header">
               </ul>
             </li>
             <li>
-              <a class="smooth-scroll" href="#media">Media</a>
-              <ul class="nav">
-                <li><a class="smooth-scroll" href="#photos">Photos</a></li>
-                <li><a class="smooth-scroll" href="#videos">Videos</a></li>
-              </ul>
+              <a class="smooth-scroll" href="#media">Photos & Videos</a>
```
[the remark this answers is not in the document]
Comment (gregmarra) [reply templates_jinja2/team_details.html]: Oops wait, fix this one thing still.

Did my comment about the ampersand encoding fix itself? It should be `& a m p ;`, not just a naked ampersand.

**Comment by gregmarra** [comment]
Accidentally approved this, please fix the ampersand thing.

The indentation stuff is fine for now – don't worry about it.

**Comment by gregmarra** [comment]
LGTM!

**Pull request merged**